### PR TITLE
Use correct service_url when calling make_document

### LINKED ISSUE #95: migrate_to_volto service_url issue
When migrate_to_volto.py reaches this line, it calls get_blocks_from_richtext without specifying service_url, which makes it always use the default value http://localhost:5000/html instead of the value the user has filled in the @@migrate_to_volto form.
https://github.com/plone/plone.volto/blob/b979e5c7e897d9b3117fece0dbe8e430b5a9503f/src/plone/volto/browser/migrate_to_volto.py#L277

### PR DESCRIPTION
**Pull request by pbauer**
Fix #95

### CODE REVIEW

**Comment by pbauer** [comment]
@jenkins-plone-org please run jobs

**Review by davisagli — APPROVED** [review]

**Comment by tisto** [comment]
@jenkins-plone-org please run jobs

File: src/plone/volto/browser/migrate_to_volto.py [hunk 1]
```diff
@@ -105,7 +105,7 @@ def do_migrate_folders(self):
         catalog = getToolByName(self.context, "portal_catalog")
         for brain in catalog(portal_type="Folder", sort_on="path"):
             obj = brain.getObject()
-            obj = make_document(obj, slate=self.slate)
+            obj = make_document(obj, service_url=self.service_url, slate=self.slate)
```
Comment (tisto) [inline src/plone/volto/browser/migrate_to_volto.py:108]: @pbauer just out of curiosity. What is service_url here?

Comment (mauritsvanrees) [reply src/plone/volto/browser/migrate_to_volto.py:108]: Today I released this in `plone.volto 4.0.8`, which is in Plone 6.0.3rc1.

**Pull request merged**